openfisca_france/parameters/bouclier_fiscal.xml

```diff
@@ -1,6 +1,7 @@
 <NODE code="bouclier_fiscal" description="Bouclier fiscal" origin="openfisca">
   <CODE code="taux" description="Taux d'imposition maximal" format="percent" origin="openfisca">
-    <VALUE deb="2007-01-01" fin="2010-12-31" valeur="0.5" /> # Suppression du bouclier fiscal à compter des revenus réalisés en 2011. #
-    <VALUE deb="2006-01-01" fin="2006-12-31" valeur="0.6" />
+    <END deb="2011-01-01" />
+    <VALUE deb="2007-01-01" valeur="0.5" /> # Suppression du bouclier fiscal à compter des revenus réalisés en 2011. #
```

Comment:
The transformation was automated. I am willing to check manually for every comment and decide to which line it naturally belongs. However, I would like to be sure that the automated regexes won't need to be modified and replayed.